src/quo/components/counter/step/view.cljs

```diff
@@ -23,6 +24,22 @@
      [text/text
       {:weight :medium
        :size   :label
-       :style  {:color (style/text-color type theme)}} label]]))
+       :style  {:color (style/text-color type theme)}}
+      label]]))
 
-(def view (theme/with-theme view-internal))
+(def ?schema
```

Comment:
maybe it could be defined above the usage? it makes it a bit more clearer if reading it the first time, but no strong opinion

Reply:
I have no preference @cammellos, and since nobody objected to your suggestion I'll apply it.